README.md

```diff
@@ -9,16 +9,9 @@ The MoveIt! Motion Planning Framework **for ROS 2.0**
 - [Get Involved](http://moveit.ros.org/documentation/contributing/)
 
 ## Milestones
-- [ ] Setup instructions
-  - [ ] Install instructions
-    - [ ] Ubuntu 18.04
-    - [ ] OS X 10.14
```

Comment:
@vmayoral my understanding is that you don't have plans to support OSX currently, right?

Reply:
@davetcoleman we discussed this internally and we're going to try to get OS X support as well. That's the initial effort at least.